subsys/bluetooth/controller/coex/Kconfig

```diff
@@ -5,8 +5,7 @@
 
 menuconfig BT_CTLR_COEX_DRIVERS
 	bool "Bluetooth Co-existence Drivers"
-	default n
-	depends on BT_CTLR
+	depends on BT_CTLR && DT_HAS_GPIO_RADIO_COEX_ENABLED
```

Comment:
`default y` and drop `CONFIG_BT_CTLR_COEX_DRIVERS=y` from `samples/bluetooth/beacon/prj-coex.conf`?

Reply:
done